### ISSUE TITLE
Preview build doesn't render immediately

### ISSUE THREAD

**Issue by alexandratran** [MetaMask/mm-docs-v2#111]
The preview build frequently displays a "page not found" immediately after publishing, and doesn't render the PR preview until 15+ minutes later. This is inconvenient for contributors and reviewers, as we can't provide a working preview link right after submitting a PR.

![Screenshot 2023-04-10 at 1 47 47 PM](https://user-images.githubusercontent.com/12214231/230996097-615eb11e-011d-45ce-af2b-a9c4a3dc4f23.png)

**Comment by BelfordZ** [comment]
Does this happen if you wait until the gh-action has commented with a link?

**Comment by alexandratran** [comment]
@BelfordZ Yes, the link is usually commented within a minute after creating the PR / pushing commits.
